addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/ExtensionWappalyzer.java

```diff
@@ -102,8 +102,12 @@ public String getName() {
             return name;
         }
 
-        static boolean getBooleanForMode(Mode mode) {
-            return Mode.EXHAUSTIVE.equals(mode);
+        @Override
+        public String toString() {
+            if (getName() == null) {
```

Comment:
Why null check?

Reply:
Probably isn't needed, I added it based on the CustomPageType handling but that's a slightly different case. Will remove.